cts/scheduler/exp/remote-connection-unrecoverable.exp

```diff
@@ -2,7 +2,7 @@
   <synapse id="0">
     <action_set>
       <pseudo_event id="9" operation="stop" operation_key="remote1_stop_0">
-        <attributes CRM_meta_timeout="20000"  reconnect_interval="60"/>
+        <attributes CRM_meta_expire_attrs="true" CRM_meta_timeout="20000"  reconnect_interval="60"/>
```

Comment:
FYI in this test, the remote connection will not be started, so it's a good test case for that